source/get-started/connect-to-mongodb.txt

```diff
@@ -0,0 +1,87 @@
+.. _kotlin-sync-connect-to-mongodb:
+
+==================
+Connect to MongoDB
+==================
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create the Application File
+    
+      Create a file called ``DemoDataClassExample.kt`` in your project.
+
+      Copy the following sample code into the file and replace the value of
+      the ``<connection URI string>`` placeholder with your MongoDB
+      Atlas connection string that you saved in the preceding step.
+
+      .. literalinclude:: /includes/get-started/DemoDataClassExample.kt
+         :language: kotlin
+         :caption: DemoDataClassExample.kt
+
+      .. note::
+      
+         This example uses a {+language+} data class to model MongoDB data.
```

Comment:
I'm seeing that we have a task to add a Specialized Data Formats > Data Classes page, could be a good idea to leave a TODO here to link to that page for more info once it's built.